daprdocs/content/en/developing-ai/dapr-agents/dapr-agents-getting-started.md

```diff
@@ -51,162 +51,326 @@ docker ps
 
-### 1. Create the Dapr components
+### 1. Clone the repository and examine its content
 
-Create a `components` directory and add two files:
+```bash
+git clone https://github.com/dapr/dapr-agents.git
+cd dapr-agents/quickstarts/01-dapr-agents-fundamentals
+```
+
+### 2. Create a virtual environment and install dependencies
+
+From the `01-dapr-agents-fundamentals` folder, do:
+
+```bash
+python3.10 -m venv .venv
+
+# Activate the virtual environment 
+# On Windows:
+.venv\Scripts\activate
+# On macOS/Linux:
+source .venv/bin/activate
+
+# Install dependencies from the quickstart
+pip install -r requirements.txt
+```
 
-`historystore.yaml`:
+This installs `dapr-agents` and any additional libraries needed by the examples.
+
+## Understand the application
+
+This example creates an agent that assists with weather information and uses Dapr to handle LLM interactions, persist conversation history, and provide reliable, durable execution of the agent’s steps.
+
+For this quickstart you’ll primarily work with:
+
+* `06_durable_agent_http.py` – the main durable weather agent application exposed over HTTP
+* `function_tools.py` – contains `slow_weather_func`, the tool used by the agent
+* `resources/llm-provider.yaml` – Conversation API and LLM configuration
+* `resources/conversation-statestore.yaml` – conversation memory state store
+* `resources/workflow-statestore.yaml` – workflow and durable execution state store
+
+
+Open `06_durable_agent_http.py`:
+
+```python
+from dapr_agents.llm import DaprChatClient
+
+from dapr_agents import DurableAgent
+from dapr_agents.agents.configs import AgentMemoryConfig, AgentStateConfig
+from dapr_agents.memory import ConversationDaprStateMemory
+from dapr_agents.storage.daprstores.stateservice import StateStoreService
+from dapr_agents.workflow.runners import AgentRunner
+from function_tools import slow_weather_func
+
+
+def main() -> None:
+    # This agent is of type durable agent where the execution is durable
+    weather_agent = DurableAgent(
+        name="WeatherAgent",
+        role="Weather Assistant",
+        instructions=["Help users with weather information"],
+        tools=[slow_weather_func],
+        # Configure this agent to use Dapr Conversation API.
+        llm=DaprChatClient(component_name="llm-provider"),
+        # Configure the agent to use Dapr State Store for conversation history.
+        memory=AgentMemoryConfig(
+            store=ConversationDaprStateMemory(
+                store_name="conversation-statestore",
+                session_id="06-durable-agent-http",
+            )
+        ),
+        # This is where the execution state is stored
+        state=AgentStateConfig(
+            store=StateStoreService(store_name="workflow-statestore"),
+        ),
+    )
+
+    # This runner will run the agent and expose it on port 8001
```

Comment:
I would love to have a longer more detailed description of what a runner can do here. in particular is this 1:1 to an agent?